ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.13-nightly-x86.yaml

```diff
@@ -93,6 +96,7 @@ tests:
       HOSTED_CP: "true"
       MULTI_AZ: "false"
       OPENSHIFT_VERSION: "4.15"
+      REGION: us-east-1
```

Comment:
i saw this one was in just a normal rosa cluster, do we need to change the rosa ones too

Reply:
No, the region changes are needed only for HCP clusters.
This reminds me, you might want to change Chaos HCP tests too.